src/main/java/teammates/ui/webapi/output/InstructorData.java

```diff
@@ -18,7 +18,7 @@
     public InstructorData(InstructorAttributes instructorAttributes) {
         this.courseId = instructorAttributes.getCourseId();
         this.email = instructorAttributes.getEmail();
-
+        this.role = instructorAttributes.getRole();
```

Comment:
I think you should hide the information to students.

Reply:
Updated